### PR TITLE
Updating nitrogen equations for compost bedded pack

### PR DESCRIPTION
**Pull request by ltp23**
This PR improves on the existing code for calculating nitrogen losses for the compost bedded pack system by reorganizing the hard-coded constants, adding more test cases and error-checking for negative nitrogen values.

### CODE REVIEW

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py
Comment (Isgotkowitz) [inline RUFAS/routines/manure/gas_emissions/gas_emissions.py:1]: It seems that most of the equations modified in this file are for calculating the gas emissions when compost bedded pack is being used. Are there other equations for computing emissions when non-CBPB bedding is used? If so, it may be good to indicate these things in the names of the methods, so it is extra clear which equations should be used for each bedding type.

File: RUFAS/routines/manure/manure_treatments/manure_treatment_cbpb.py [hunk 1]
```diff
@@ -66,14 +67,26 @@ def _calc_bedding_potassium_content(
...
 
         """
-        daily_output = self._initialize_daily_output_during_update(self._current_manure_treatment_daily_input)
+        daily_input = self._current_manure_treatment_daily_input
+        total_nitrogen_loss = GasEmissions.calc_total_nitrogen_loss_from_compost_bedded_pack_barn(
+            daily_nitrogen_input=daily_input.liquid_manure_nitrogen,
+            is_bedding_tilled=True
+        )
+        manure_nitrogen = daily_input.liquid_manure_nitrogen - total_nitrogen_loss
+        # TODO: To be implemented further later.
```
Comment (Isgotkowitz) [inline RUFAS/routines/manure/manure_treatments/manure_treatment_cbpb.py:86]: Dangling TODO

File: tests/test_manure_module/test_gas_emissions.py
Comment (Isgotkowitz) [inline tests/test_manure_module/test_gas_emissions.py:1]: These test cases look great, but I think that "Act and Assert" and "Arrange" in-line comments are unnecessary.

**Review by Isgotkowitz — APPROVED** [review]
Code looks great, 1 comment about the naming of methods specific to CBPB treatment method, 2 comments on other comments in the code.

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py
Comment (ltp23) [reply RUFAS/routines/manure/gas_emissions/gas_emissions.py:1]: That sounds good. I will update the names of those functions to be more specific.

File: RUFAS/routines/manure/manure_treatments/manure_treatment_cbpb.py (hunk 1 above)
Comment (ltp23) [reply RUFAS/routines/manure/manure_treatments/manure_treatment_cbpb.py:86]: Yes, I need to update this method incrementally.

File: tests/test_manure_module/test_gas_emissions.py
Comment (ltp23) [reply tests/test_manure_module/test_gas_emissions.py:1]: Those are entirely optional. I keep those to remind myself of the structure of each test.

**Review by varrma — APPROVED** [review]
